### PR TITLE
feat: Bundle analyzer

### PR DESCRIPTION
**Pull request by info-arnav**
Reference #1838 

**Webpack Bundle Analyzer**

The pull request doesnt itself offer any changes yet, just a template for the change I am planning.

I plan on creating an own bundle analyzer for webpack refering to [https://github.com/webpack-contrib/webpack-bundle-analyzer](url)

Converting it to typeScript, using webpack modules such as logger and caching and refactoring everything a bit.

Please just let me know if the approach is fine, if yes ill continue with writing the program.

### CODE REVIEW

**Comment by info-arnav** [comment]
> Reference #1838
> 
> **Webpack Bundle Analyzer**
> 
> The pull request doesnt itself offer any changes yet, just a template for the change I am planning.
> 
> I plan on creating an own bundle analyzer for webpack refering to [https://github.com/webpack-contrib/webpack-bundle-analyzer](url)
> 
> Converting it to typeScript, using webpack modules such as logger and caching and refactoring everything a bit.
> 
> Please just let me know if the approach is fine, if yes ill continue with writing the program.

@alexander-akait @evenstensberg

File: packages/webpack-cli/src/webpack-cli.ts [hunk 1]
```diff
@@ -985,20 +986,6 @@ class WebpackCLI implements IWebpackCLI {
...
-        configs: [
-          {
-            type: "enum",
-            values: [true],
-          },
-        ],
-        multiple: false,
-        description: "It invokes webpack-bundle-analyzer plugin to get bundle information.",
-        helpLevel: "minimum",
-      },
```
Comment (snitin315) [inline packages/webpack-cli/src/webpack-cli.ts:1001]: Why was this removed?

**Review by snitin315 — CHANGES_REQUESTED** [review]
From https://github.com/webpack/webpack-cli/issues/1838#issuecomment-699056629, only the last action item is pending.

>need implement schema for webpack-bundle-analyzer and read options from schema (can be done late)

File: packages/webpack-cli/src/webpack-cli.ts (hunk 1 above)
Comment (info-arnav) [reply packages/webpack-cli/src/webpack-cli.ts:1001]: I was thinking of a custom plugin, so was planning on changing stuff accordingly. But got it, will start working on the last action.

**Comment by alexander-akait** [comment]
Thank issue should be resolved on webpack-bundle-analyzer, you need to add such stuff https://github.com/webpack/webpack-dev-server/blob/master/bin/cli-flags.js, so webpack-cli reads all CLI flags and provide their to user

**Comment by info-arnav** [comment]
will start today
